diagnostic.md

```diff
@@ -35,5 +41,5 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{my-phone passedData='passedData'}}
```

Comment:
To load the nested component and pass it data, we need to iterate through all of the phone numbers, then we need to access the object property, last we need to bind the sub-component.
```
{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}
```